src/Runner.Listener/BrokerMessageListener.cs

```diff
@@ -432,17 +433,10 @@ ex is AccessDeniedException ||
 
         private async Task RefreshBrokerConnectionAsync()
         {
-            var configManager = HostContext.GetService<IConfigurationManager>();
-            _settings = configManager.LoadSettings();
-
-            if (string.IsNullOrEmpty(_settings.ServerUrlV2))
```

Comment:
we don't need to check this again, we are in BrokerMessageListener only if we have the ServerUrlV2 in the first place.